superset-frontend/src/components/Label/index.tsx

```diff
@@ -93,6 +93,7 @@ export default function Label(props: LabelProps) {
   return (
     <Tag
       onClick={onClick}
+      role={onClick ? 'button' : undefined}
```

Comment:
We should move this logic inside the Tag component

Reply:
Ok I see this `Tag` is pulled directly from Antdesign rather than our custom `Tag` component, so it makes sense to keep it here